sdk/storage/Azure.Storage.Files.DataLake/README.md

```diff
@@ -33,39 +34,178 @@ az storage account create --name MyStorageAccount --resource-group MyResourceGro
 
 ## Key concepts
 
-TODO: Add Key Concepts
+This preview package for .NET includes ADLS Gen2 specific API support made available in Blob SDK. This includes:
```

Comment:
This section is meant to convey key DataLake concepts and this reads more like release notes.  I would expect this section to contain a bullet point view of the data in https://docs.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-introduction (like Blobs/Files.Shares/etc.).

Reply:
`in Blob SDK` should say something more like `Azure Storage Blobs` if you're going to leave this section as-is.  As far as our official branding goes, there's no Blob SDK.  There's an Azure.Storage.Blobs client library in the .NET SDK.

Reply:
Did you want me to make a summary of that introduction, or just link to that introduction?

Reply:
I'd go for a summary.  Here's the [Blobs README](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/storage/Azure.Storage.Blobs#key-concepts) and here's [where we stole the key concepts from](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction).